lib/parser.dart

```diff
@@ -485,7 +485,11 @@ class _Parser {
    *  mixin:              '@mixin name [(args,...)] '{' declarations/ruleset '}'
    *  include:            '@include name [(@arg,@arg1)]
    *                      '@include name [(@arg...)]
-   *  content             '@content'
+   *  content:            '@content'
+   *  -moz-document:      '@-moz-document' [ <url> | url-prefix(<string>) |
+   *                          domain(<string>) | regexp(<string) ]# '{'
+   *                        declarations
+   *                      '}'
```

Comment:
Might consider support @document too its the replacement of -moz-document and there you could enforce the quotes for <string>.  Always thought it would be good to generate warnings for deprecated syntax as new CSS syntax becomes available (at least for input) the emitted could be old style to be browser agnostic until a browser supports the better syntax.